plugin/teksi_wastewater/metadata.txt

```diff
@@ -1,13 +1,14 @@
 
 [general]
 name=TEKSI Wastewater
-description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools.
+description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools. Require a compatible TEKSI wastewater datamodel and project file. Support Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK (2020.x) for INTERLIS import / export.
```

Comment:
description=TEKSI Wastewater plugin to manage wastewater networks. Provides selection, connection and editing tools. Requires a compatible TEKSI wastewater datamodel and project file. Supports Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK (2020.x) for INTERLIS import / export.

Reply:
Thanks @ponceta